src/styles/animations.scss

```diff
@@ -0,0 +1,9 @@
+@keyframes rotation {
```

Comment:
Consider adding separate animations for minute and second hands to control their rotation speeds independently

Reply:
thank you for this comment, but I've created this animation for 2 stopwatch hands, I know, that hands will rotate from 0deg to 360deg. Would it make sense to leave it like it is, because this elements do same moves?